frontend/src/component/feature/FeatureStrategy/FeatureStrategyEmpty/AddFromTemplateCard/AddFromTemplateCard.tsx

```diff
@@ -0,0 +1,89 @@
+import { ElementType, FC } from 'react';
+import { ChangeRequestDialogue } from 'component/changeRequest/ChangeRequestConfirmDialog/ChangeRequestConfirmDialog';
+import useFeatureStrategyApi from 'hooks/api/actions/useFeatureStrategyApi/useFeatureStrategyApi';
+import useToast from 'hooks/useToast';
+import { AddStrategyMessage } from 'component/changeRequest/ChangeRequestConfirmDialog/ChangeRequestMessages/AddStrategyMessage';
+import { useChangeRequestsEnabled } from 'hooks/useChangeRequestsEnabled';
+import { useChangeRequestAddStrategy } from 'hooks/useChangeRequestAddStrategy';
+import { formatUnknownError } from 'utils/formatUnknownError';
+import { useFeature } from 'hooks/api/getters/useFeature/useFeature';
+import { PresetCard } from './PresetCard/PresetCard';
+import { IFeatureStrategyPayload } from 'interfaces/strategy';
+
+interface IAddFromTemplateCardProps {
+    title: string;
+    featureId: string;
+    projectId: string;
+    environmentId: string;
+    strategy: IFeatureStrategyPayload;
+    Icon: ElementType;
+    onAfterAddStrategy: () => void;
+}
+
+export const AddFromTemplateCard: FC<IAddFromTemplateCardProps> = ({
```

Comment:
I'd consider adding PresetCard as {children} to avoid prop-drilling title, children, stretegy, Icon, onAfterAddStrategy. More info about this technique: https://www.youtube.com/watch?v=3XaXKiXtNjw&ab_channel=ReactTraining

Reply:
I got rid of PresetCard, embedded the code in parent. At this point it was just styling